dotCMS/src/main/java/com/dotcms/keyvalue/model/KeyValue.java

```diff
@@ -0,0 +1,116 @@
+package com.dotcms.keyvalue.model;
+
+import java.io.Serializable;
+import java.util.Map;
+
+import com.dotcms.contenttype.model.type.MultilinguableFallback;
+import com.dotmarketing.business.DotStateException;
+import com.dotmarketing.business.Permissionable;
+import com.dotmarketing.business.Ruleable;
+import com.dotmarketing.business.Treeable;
+import com.dotmarketing.business.Versionable;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.util.Config;
+
+/**
+ * Represents a content of type Key/Value in the system. These types of contents are very useful for
+ * both inner workings of dotCMS and custom content. For example, Language Variables are a sub-type
+ * of Key/Value objects.
+ * 
+ * @author Jose Castro
+ * @version 4.2.0
+ * @since Jun 19, 2017
+ *
+ */
+public interface KeyValue extends Serializable, Versionable, Permissionable, Treeable, Ruleable, MultilinguableFallback {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)